lib/api/channel.request.js

```diff
@@ -7,17 +7,17 @@ class Channel extends Request {
     super(httpClient)
   }
 
-  get(channelId, accessToken) {
+  get(channelId) {
     debug('get() channel', channelId)
-    const headers = this.createHeaders(accessToken)
+    const headers = this.createHeaders(arguments)
```

Comment:
Všude předělej `createHeaders()` na `_createHeaders()`. To "**_**" je konvence z pythonu akceptovaná i v jiných jazycích a říká "použij mě jen na vlastní nebezpečí. Já jsem privátní/protected funkce".

Reply:
:+1: